packs/clicrud/config.yaml

```diff
@@ -0,0 +1,5 @@
+---
+  username: "admin"
```

Comment:
If you get a chance it would also be great to add config.schema.yaml file - https://docs.stackstorm.com/reference/pack_configs.html

This way operator / admin can also specify password as in a datastore as a secret.